keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/ActuationPauser.kt

```diff
@@ -92,44 +89,4 @@ class ActuationPauser(
 
   fun pausedApplications(): List<String> =
     pausedRepository.getPausedApplications()
-
-  /**
-   * Adds [ApplicationActuationPaused] and [ApplicationActuationResumed] events to the resource event history so that
-   * it reflects actuation pauses/resumes at the application level.
-   */
-  fun addApplicationActuationEvents(originalEvents: List<ResourceEvent>, resource: Resource<*>) =
```

Comment:
All this hacky stuff is now gone as we simply retrieve the right events from the data store.

Reply:
That's a relief!